src/main/kotlin/com/exactpro/th2/httpserver/server/options/ServerOptions.kt

```diff
@@ -43,9 +43,7 @@ interface ServerOptions {
      */
     fun createExecutorService(): ExecutorService
 
-    fun <T:RawHttpResponse<*>> prepareResponse(request: RawHttpRequest, response: T): T = response
-
     fun onRequest(request: RawHttpRequest, id: String) = Unit
 
-    fun <T:RawHttpResponse<*>> onResponse(request: RawHttpRequest, response: T) = Unit
+    fun <T : RawHttpResponse<*>> onResponse(request: RawHttpRequest, response: T) = response
```

Comment:
```suggestion
    fun <T : RawHttpResponse<*>> prepareResponse(request: RawHttpRequest, response: T) = response
```

We should also state in interface docs that it should be implemented in thread-safe manner since its methods will be called from different threads